scss/bitstyles/settings/shadows.stories.mdx

```diff
@@ -0,0 +1,145 @@
+
+## Using shadows
+
+The accessor function `shadows.get()` is the most straightfoward way to use shadows stored in this format. Specify the name of the shadow and the output format you need (these examples are taken from the settings files for `box-shadow` and `drop-shadow` utility classes, each of which uses a different format):
+
+```scss
+@use '../../tools/shadows';
+
+$bitstyles-box-shadow-values: (
+  'default': shadows.get('default', 'css'),
+);
+
+/*
+
+box-shadow:
+  0 0.5rem 0.5rem rgba(0, 13, 255, 0.15),
+  0 0.6666666667rem 0.6666666667rem rgba(0, 13, 255, 0.15),
+  0 1rem 1rem rgba(0, 13, 255, 0.15),
+  0 2rem 2rem rgba(0, 13, 255, 0.15);
+
+*/
+
+$bitstyles-drop-shadow-values: (
+  'default': shadows.get('default', 'filter'),
+);
+
+/*
+
+filter:
+  drop-shadow(rgba(0, 13, 255, 0.15) 0 0.5rem 0.5rem)
+  drop-shadow(rgba(0, 13, 255, 0.15) 0 0.6666666667rem 0.6666666667rem)
+  drop-shadow(rgba(0, 13, 255, 0.15) 0 1rem 1rem)
+  drop-shadow(rgba(0, 13, 255, 0.15) 0 2rem 2rem);
+
+*/
+```
+
+### Using local shadows
+
+Not all shadows belong among the global shadows design tokens. Including all the shadows applied to buttons and input elements on hover/focus/active/disabled/etc. could swamp the global list. But you may want to use `shadow.generate()` to take advantage of the multi-layered shadows without having to hand-code them. You can do this using one of the two helper functions `shadow.to-css()` and `shadow.to-filter()`. These convert the list formatted-shadows provided by the generator into a format suitable for use in CSS `box-shadow` properties and CSS `filter` properties (using `drop-shadow`), respectively.
+
+For example, take the two shadows overriding the `box-shadow` and `drop-shadow` settings above. We could specify them in-place instead of `shadow.get()`ing them, if they were not part of our design system right now:
+
+```scss
+@use '../../tools/shadow';
+
+$bitstyles-box-shadow-values: (
+  'default': shadow.to-css(
+    shadow.generate(
+      $color: rgba(palette.get('brand-1'), 0.15),
+      $offset: 2rem,
+      $blur-radius: 2rem,
+    )
+  ),
+);
+
+$bitstyles-drop-shadow-values: (
+  'default': shadow.to-filter(
+    shadow.generate(
+      $color: rgba(palette.get('brand-1'), 0.15),
+      $offset: 2rem,
+      $blur-radius: 2rem,
+    )
+  ),
+);
+
+```
+
+## Specifying shadows manually
+
+You may need more fine-grained control of the shadows — for example when your designs differ from the shadows the generator produces. You can use bitstyles’ central shadow settings without using `shadow.generate()`, but it means writing the format by hand.
+
+In this format, each shadow is specified as a list of maps (Sass lists and Sass maps. This can be thought of as similar to an array of objects in javascript). Each map represents a layer of the shadow, and has the `color`, `offset`, and `blur-radius` keys as defined in the parameters for `generate()`, shown in the table above.
```

Comment:
```suggestion
In this format, each shadow is specified as a list of maps (Sass lists and Sass maps. This can be thought of as similar to an array of objects in JavaScript). Each map represents a layer of the shadow, and has the `color`, `offset` and `blur-radius` keys as defined in the parameters for `generate()`, shown in the table above.
```

Reply:
I was wondering if that would be considered too informal xD